dcrlibwallet.go

```diff
@@ -2,70 +2,63 @@ package dcrlibwallet
 
 import (
 	"context"
-	"fmt"
-	"os"
 	"path/filepath"
 
-	"github.com/asdine/storm"
-	"github.com/decred/dcrwallet/errors"
-	"github.com/decred/dcrwallet/netparams"
-	"github.com/decred/dcrwallet/wallet"
-	"github.com/decred/dcrwallet/wallet/txrules"
+	"github.com/decred/dcrwallet/errors/v2"
+	wallet "github.com/decred/dcrwallet/wallet/v3"
+	"github.com/decred/dcrwallet/wallet/v3/txrules"
+	l "github.com/raedahgroup/dcrlibwallet/internal/loader"
+	"github.com/raedahgroup/dcrlibwallet/internal/netparams"
 	"github.com/raedahgroup/dcrlibwallet/txindex"
 	"github.com/raedahgroup/dcrlibwallet/utils"
-	"go.etcd.io/bbolt"
 )
 
-const logFileName = "dcrlibwallet.log"
+const (
+	SpendingPassphraseTypePin  int32 = 0
+	SpendingPassphraseTypePass int32 = 1
+)
+
+type WalletProperties struct {
+	WalletID               int    `storm:"id,increment"`
+	WalletName             string `storm:"unique"`
+	WalletDataDir          string
+	WalletSeed             string
+	DefaultAccount         int32
+	SpendingPassphraseType int32
+	DiscoveredAccounts     bool
+}
 
 type LibWallet struct {
-	walletDataDir string
-	activeNet     *netparams.Params
-	walletLoader  *WalletLoader
-	wallet        *wallet.Wallet
-	txDB          *txindex.DB
-	configDB      *storm.DB
-	*syncData
+	WalletProperties `storm:"inline"`
```

Comment:
Here's a thought. Can we save only `WalletProperties` and not `LibWallet`s?

Reply:
https://github.com/c-ollins/mobilewallet/commit/8939ee09407dacbc4332dd90b5183094fc275d48#diff-31ceaeff2e093812f35876b043453fc0R133